src/docs/wasm-compilation-pipeline.md

```diff
@@ -44,14 +44,14 @@ For experimentation, V8 and Chrome can be configured to compile WebAssembly code
 - Lazy compilation:
     - Lazy compilation is a compilation mode where a function is only compiled when it is called for the first time. Similar to the production configuration the function is first compiled with Liftoff (blocking execution). After Liftoff compilation finishes, the function gets recompiled with TurboFan in the background.
     - In V8, set the `--wasm-lazy-compilation` flag.
-    - In Chrome, enable WebAssembly lazy compilation (chrome://flags/#enable-webassembly-lazy-compilation).
+    - In Chrome, enable [WebAssembly lazy compilation](chrome://flags/#enable-webassembly-lazy-compilation).
```

Comment:
I think, like with `chrome://tracing`, this is not actually linkable? (is it?)

Reply:
Yeah, just checked and it's non-clickable. Let's format it with backticks like the other one.

Reply:
Done.